virtcontainers/qemu_arm64.go

```diff
@@ -40,6 +40,7 @@ var kernelParams = []Param{
 	{"console", "hvc0"},
 	{"console", "hvc1"},
 	{"iommu.passthrough", "0"},
+	{"printk.time", "1"},
```

Comment:
ack. Alternatively, you could set `PRINTK_CONFIG_TIME` in the kernel config file at https://github.com/kata-containers/packaging/blob/master/kernel/configs/arm64_kata_kvm_4.19.x#L2277 ? That way we shift the decision to print time or not to the kernel builder, and not permanently enabled on the qemu command line.

/me notes, might be nice to consider moving to the [kernel config fragments setup](https://github.com/kata-containers/packaging/tree/master/kernel/configs/fragments) one day?

Reply:
OK, I will enable printk_time in kernel/config and this pr can be closed. thanks @grahamwhaley

Reply:
Ping @jongwu